src/platform/wearables/wearable.ts

```diff
@@ -0,0 +1,94 @@
+import { generateValidator, JSONSchema, ValidateFunction } from '../../validation'
+import { Rarity } from '../../dapps/rarity'
+import { WearableCategory } from '../../dapps/wearable-category'
+import { I18N } from './i18n'
+import { WearableRepresentation } from './representation'
+import { Metrics } from './metrics'
+
+export type Wearable = {
+  id: string
+  description: string
+  collectionAddress: string
+  rarity: Rarity
+  i18n: I18N,
```

Comment:
As we are moving towards standardizing this schema for third parties too. Maybe we will need to change `i18n` to have different locales for the `description` field too. 
```ts
{
code: Locale
values: { key: text,  value: text }[]
}
```

What do you think? @nchamo @menduz ?

Reply:
Sounds like good idea, but that would require work on both the catalysts (to transform current strings to i18n) and the explorer (to start supporting i18n).

I'll let @pentreathm decide if there is enough bandwidth to support the change

Reply:
@nachomazzara can we take on that task as part of the third party werables initiative so that we can close this? as it is a request that requires changes in several places I would take that in a different place.

Reply:
Just to clarify. In my opinion, the best solution would be to:
1. Replace the `i18n: I18N[]` property for `name: I18N[]`
2. Replace the `description: string` property for `description: I18N[]`

Reply:
@nachomazzara , talked to @pentreathm offline and we decided to go ahead with defining the "final" version of the schemas, but we will wait to start using them for validation.

Reply:
The only advantage related to having everything inside `i18n` that I'm seeing is to have dynamic text properties. But not sure what else we can add there. @nicosantangelo  Do you remember why we named it `i18n` instead of `name: I18N[]` ?

Reply:
My two cents, to minimize conflicts, if you change the types of properties, the properties are different, therefore should be called differently.

`description: string` may be called `descriptionS: array<i18n>`

it is a rule of thumb, not specific for this changes. but fwow, if we are already using `name:string` I'd keep `i18n: I18N` for the name, and make name optional

Reply:
I wouldn't use an array for that as that could imply having more than one value for the same Locale key, I would use a map instead. Also, I'd add validation to be sure that only supported locales are keys.

Something like this:

```
...

name: {
   EN: "John",
   ES:  "Juan"
},
description: {
  EN: "This is a description",
  ES: "Esta es una descripción"
}

...
```


Or, if we want to have all the translations on the same object I would use be something like:

```
...

i18n: {
  name: {
    EN: "John",
    ES: "Juan"
  },
  description: {
    EN: "This is a description",
    ES: "Esta es una descripción"
  }
}

...
```

Anyway, in any of both cases, I'd validate that the keys for locale are one of the supported ones, and in the second case I'd validate that any field inside i18n is valid too. This implies that we will need to change the schema to support any new locale.